test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -1311,13 +1311,15 @@ TEST_F(PickFirstTest, PendingUpdateAndSelectedSubchannelFails) {
       BuildChannel("", response_generator);  // pick_first is the default.
   auto stub = BuildStub(channel);
   // Create a number of servers, but only start 1 of them.
-  CreateServers(10);
+  CreateServers(2);
```

Comment:
I think you can replace these two lines with just `StartServers(2)`.  Now that we're using the connection attempt injector, we can start both servers from the get-go, and we can just release the hold when we're ready for the second server to get connected.

Reply:
Done